GameDefinition/Content/ItemKind.hs

```diff
@@ -1739,3 +1739,24 @@ currency = currencyTemplate
                ++ iaspects currencyTemplate
   , ieffects = [RefillCalm (-1)]
   }
+  
+-- * Clothing
+
+smokingJacket = ItemKind
+  { isymbol  = _symbolClothes
+  , iname    = "smoking jacket"
+  , ifreq    = [("common item", 100), ("misc clothing", 1)]
+  , iflavour = zipPlain [Green]
```

Comment:
```suggestion
  , iflavour = zipFancy [Green]
```
Unfortunately, currently the fancy-flavoured name for `Green` is "forest-green". But if it doesn't match your vision of the jacket, we may create another set of flavours, say `Natural` and move "forest-green" there and here substitute something really fancy. We need another set or flavours anyway. Proposals?

https://github.com/LambdaHack/LambdaHack/blob/master/Game/LambdaHack/Common/Flavour.hs#L102

Reply:
BrGreen works - that would make it an emerald smoking jacket, which implies the appropriate richness.

Reply:
Perfect.